README.md

```diff
@@ -1,2 +1,109 @@
+
+Project was firstly developed during [XRPL NFT hackathon.](https://devpost.com/software/xrp-nft-attendence)
+
+## Features
+
+⚫ Creating new events and batch minting NFTs for it
+
+⚫ Uploading NFT metadata to IPFS
+
+⚫ Checking if it's possible to claim NFTs for particular event
+
+⚫ Requesting claim for NFT from particular event. The new sell offer for NFT that has to be accepted by the user is returned
+
+⚫ Event attendees lookup
+
+⚫ NFT ownership verification
+
+## Documentation
+
+### Attendify.js
+
+⚫ Current version of documentation for Attendify.js is in the `docs` folder and was generated by jsdoc. Latest version of docs is hosted [here](https://justanotherdevv.github.io/AttendifyXRPL/Attendify.html).
+
+### REST API endpoints
+
+This API is built using the ExpressJS framework and the XRPL library, it connect to the local Attendify.js class to perform most of the actions. 
+
+⚫ `GET /api/mint` - Creates new claimable event and premints NFTs for it. You must have at least X XRP in order to use this to handle the reserve requirements for the NFTs and the minting process which uses [Tickets](https://xrpl.org/tickets.html), where X is amount of tokens that should be minted multiplied by 2 + additional XRP for feees.
+
+**@param {string} walletAddress** - Wallet address from user requesting claim
+
+**@param {integer} tokenCount** - Amount of NFTs that should be create for the event
+
+**@param {string} url** - Url to Image for event | Preferably this should be stored on IPFS
+
+**@param {string} title** - Title for event
+
+**@param {string} desc** - Description of event
+
+**@param {string} loc** - Location of event
+
+⚫ `GET /api/claim` - Checks if user is eligible for NFT claim or creates new offer that allows for claiming NFT for specific event. Endpoint returns response with sellOfferID that has to be accepted by user afterwards.
+
+**@param {string} walletAddress** - The wallet address of the user trying to claim
+
+**@param {integer} type** - The type of claim (1 for checking claim status and metadata, 2 for claiming)
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+**Possible responses**
+
+    -`404` - Event with specified ID does not exist
+
+    -`claimed` - The NFT for specified event was already claimed by this user
+
+    -`empty` - All NFTs for this particular event were already claimed
+
+    -`success` - If `onlyCheckStatus` parameter was true it indicated that current parameters for selected event were sent
+
+    -`transferred` - Indicates that new sell offer for NFT related to selected event was created successfully and details were sent to user
+
+⚫ `GET /api/verifyOwnership` - Verifies whether or not user owns NFT with provided id for particular user.
+
+**@param {string} walletAddress** - Wallet address from user requesting verification
+
+**@param {string} signature** - Signature from requesting user
```

Comment:
What do they have to sign? I'm a bit unclear what I'm supposed to put into this parameter. (Is it a fake transaction like XUMM does?)

Reply:
(If it can be any transaction I'd recommend expanding this description to include an example of what you could sign - Ex. An `AccountSet` transaction with `SetFlag` set to 0 (no flags) is a common useless operation you can have people sign)

Reply:
Yeah, it's just a signature to prove ownership of `walletAddress`. In integration with UI this could be done using XUMM, currently for the purpose of testing I am firstly getting user wallet `const myWallet = xrpl.Wallet.fromSeed(walletForSignatureVerification.seed);` and then using function from xrpl.js to create signature 
`let my_seq = 21404872;
    const txJSON = {
      Account: myWallet.address,
      TransactionType: "Payment",
      Destination: "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
      Amount: "13000000",
      Flags: 2147483648,
      LastLedgerSequence: 7835923,
      Fee: "13",
      Sequence: my_seq,
    };
    const signature = await myWallet.sign(txJSON);`